lib/pbench/test/unit/server/test_cache_manager.py

```diff
@@ -913,17 +893,11 @@ def test_filestream(
         tar = Path("/mock/dir_name.tar.xz")
         cache = Path("/mock/.cache")
 
-        def mock_extract(tarball_path, path):
-            return exp_stream
-
-        def mock_open(self, mode="rb"):
-            return exp_stream
-
         with monkeypatch.context() as m:
             m.setattr(Tarball, "__init__", TestCacheManager.MockTarball.__init__)
             m.setattr(Controller, "__init__", TestCacheManager.MockController.__init__)
-            m.setattr(Tarball, "extract", mock_extract)
-            m.setattr(Path, "open", mock_open)
+            m.setattr(Tarball, "extract", lambda _s, _t: exp_stream)
+            m.setattr(Path, "open", lambda _s, _m: exp_stream)
```

Comment:
Nice.  🙂

However, the arguments to the lambda in line 899 aren't mnemonic -- it would be better if they were something like `_t, _p` -- and the second argument to the lambda at line 900 has lost its default value -- it should be more like `_s, _m="rb"`.